extensions/amp-list/0.1/amp-list.js

```diff
@@ -387,15 +390,41 @@ export class AmpList extends AMP.BaseElement {
           AmpEvents.DOM_UPDATE, /* detail */ null, {bubbles: true});
       this.container_.dispatchEvent(event);
 
-      // Change height if needed.
-      this.measureElement(() => {
-        const scrollHeight = this.container_./*OK*/scrollHeight;
-        const height = this.element./*OK*/offsetHeight;
-        if (scrollHeight > height) {
-          this.attemptChangeHeight(scrollHeight).catch(() => {});
-        }
-      });
+      if (!isExperimentOn(this.win, 'amp-list-resize')) {
+        // Change height if needed.
+        this.measureElement(() => {
+          const scrollHeight = this.container_./*OK*/scrollHeight;
+          const height = this.element./*OK*/offsetHeight;
+          if (scrollHeight > height) {
+            this.attemptChangeHeight(scrollHeight).catch(() => {});
+          }
+        });
+      }
     }, this.container_);
+
+    if (isExperimentOn(this.getWin(), 'amp-list-resize')) {
```

Comment:
We only want to run this once right? Also, let's extract to local var to de-dupe with above.

Reply:
Done.